packages/core/src/components/tag/_overrides.scss

```diff
@@ -1,36 +1,261 @@
-.#{$ns}-dark .#{$ns}-tag {
-  color: $white1;
-  font-weight: 400;
-  font-size: 14px;
+@mixin custom-tag-kind($kind, $color: white, $background-color: transparent, $border: transparent) {
+  &.#{$ns}-kind-g-#{$kind} {
+    color: $color;
+    background-color: $background-color;
+    box-shadow: inset 0 0 0 1px $border;
+  }
 }
 
-.#{$ns}-tag .#{$ns}-large, .#{$ns}-tag.#{$ns}-large {
+.#{$ns}-tag {
+  color: $g-grey1;
+
+  padding: 4px 8px;
+  font-size: $pt-font-size;
+
+  // For multiline tags
+  >span>p {
+    color: $g-grey1;
+    font-size: $pt-font-size-small;
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  // Custom class to have the icons and tag-remove on the top:
+  &.#{$ns}-g-fixTop {
+    align-items: stretch;
+  }
+
+  .#{$ns}-dark & {
+    color: $g-dark-grey1;
+
+    // For multiline tags
+    >span>p {
+      color: $g-dark-grey1;
+      font-size: $pt-font-size-small;
+    }
+  }
+
+  // Interactive Tag:
+  &.#{$ns}-interactive {
+    cursor: pointer;
+
+    &:hover {
+      background-color: $g-base4;
+    }
+
+    &.#{$ns}-active,
+    &:active {
+      background-color: $g-base3;
+    }
+
+    &.#{$ns}-g-disabled {
+      cursor: not-allowed;
+      background-color: $g-base3;
+      color: $g-grey4;
+    }
+
+    .#{$ns}-dark & {
+      &:hover {
+        background-color: $g-dark-base4;
+      }
+
+      &.#{$ns}-active,
+      &:active {
+        background-color: $g-dark-base3;
+      }
+
+      &.#{$ns}-g-disabled {
+        background-color: $g-dark-base3;
+        color: $g-dark-grey3;
+      }
+    }
+  }
+
+  // Lozenge
+  &.#{$ns}-lozenge {
+    border-radius: 8px;
+    font-size: $pt-font-size-extra-small;
+    text-transform: uppercase;
+
+    color: $g-black;
+    background-color: $g-base1;
+
+    // Custom kinds light theme:
+    @include custom-tag-kind(folder, $g-wizard-23-700, $g-wizard-23-200);
+    @include custom-tag-kind(insight, $g-wizard-26-700, $g-wizard-26-200);
+    @include custom-tag-kind(publish, $g-water-700, $g-water-200);
+    @include custom-tag-kind(project, $g-wizard-18-700, $g-wizard-18-200);
+    @include custom-tag-kind(danger, $g-error-700, $g-error-200);
+    @include custom-tag-kind(updated, $g-grass-700, $g-grass-200);
+    @include custom-tag-kind('active', $g-avocado-700, $g-avocado-200);
+    @include custom-tag-kind(inRecipe, $g-wizard-4-700, $g-wizard-4-200);
+    @include custom-tag-kind(notInRecipe, $g-wood-700, $g-wood-200);
+    @include custom-tag-kind(recreate, $g-water-700, $g-water-200);
+    @include custom-tag-kind(pending, $g-grey1, $g-base2);
```

Comment:
I suggest to change the `kind` by `color` and use the name of the color for the values. Check the design system, it is updated with new colors (no use of colors named `wizard_xxx`)

Reply:
Updated!